Makefile

```diff
@@ -4,7 +4,7 @@ INTEGREATLY_VERSION=1.7.0
 AMQSTREAMS_VERSION=1.1.0
 RHSSO_VERSION=1.8.2
 CODEREADY_VERSION=1.2.2
-FUSE_VERSION=1.7.0
+FUSE_VERSION=7.4.0
```

Comment:
Should this be renamed to FUSE_ONLINE_VERSION or FUSEONLINE_VERSION also? One of the most confusing things about what we did in the installation repo was start to use "fuse" when we were really talking about "fuse online".

Reply:
Sure, I'll update this